sbom/sbom.go

```diff
@@ -0,0 +1,15 @@
+package sbom
+
+import (
+	"context"
+	"io"
+
+	"github.com/quay/claircore"
+)
+
+// Encoder is an interface to convert a claircore.IndexReport and writes it to
+// w.
+// that contains a Software Bill of Materials representation.
```

Comment:
nit: I think this is a mix of the old version and the new version of this comment?

Reply:
🤦 I didn't mean to add this to that commit. Should be fixed in the latest one + all the other godoc changes.